### PR TITLE
updating logic to work with the new config file

### PR DESCRIPTION
**Pull request by jeffctr**

### CODE REVIEW

File: config/config.schema.json [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+            "type": "string",
+            "enum": [
+              "GET",
+              "POST",
+              "DELETE"
+            ]
+          },
+          "exec": {
+            "type": "object",
+            "description": "The command to be executed in login host. Note that the paramters defined in {} needs to be substitued with paramers supplied to this command, and server",
```
Comment (edmondchuc) [inline config/config.schema.json]: - paramters -> parameters
- substitued -> substituted
- paramers -> parameters

Comment (jeffctr) [reply config/config.schema.json]: lol thank you but that wasn't written by me hahahah I and didn't check

File: src/resource_server/settings.py [hunk 2]
```diff
@@ -12,5 +12,6 @@
 MASTER_NODE_USER = os.environ["MASTER_NODE_USER_ENV"]
 MASTER_NODE_PORT = os.environ["MASTER_NODE_PORT_ENV"]
 
-COMMANDS_JSON_FILE = os.environ["CMD_JSON_FILE_ABS_PATH_ENV"]
+COMMANDS_JSON_FILE = f"{os.path.abspath('./')}/config/config.json"
+COMMANDS_JSON_SCHEMA = f"{os.path.abspath('./')}/config/config.schema.json"
```
Comment (edmondchuc) [inline src/resource_server/settings.py]: May I suggest pathlib?

```python
import pathlib

COMMANDS_JSON_FILE = str(pathlib.Path.cwd() / "config/config.json")
COMMANDS_JSON_SCHEMA = str(pathlib.Path.cwd() / "config/config.schema.json")
```

Comment (jeffctr) [reply src/resource_server/settings.py]: ok, will have a look.

**Pull request merged**